compatibility-verifier/sample-test-suite/config/feature-test-3-realtime.json

```diff
@@ -0,0 +1,63 @@
+{
+  "tableIndexConfig": {
+    "aggregateMetrics": false,
+    "autoGeneratedInvertedIndex": false,
+    "createInvertedIndexDuringSegmentGeneration": false,
+    "enableDefaultStarTree": false,
+    "enableDynamicStarTreeCreation": false,
+    "loadMode": "MMAP",
+    "noDictionaryColumns": ["stringDimSV2","intMetric1", "longMetric1", "floatMetric1", "doubleMetric1", "bytesMetric1"],
+    "nullHandlingEnabled": false,
+    "segmentFormatVersion": "v3",
+    "bloomFilterColumns": ["stringDimSV1"],
+    "rangeIndexColumns": ["doubleDimSV1"],
+    "rangeIndexVersion": 2,
+    "varLengthDictionaryColumns": ["stringDimSV1"],
+    "segmentPartitionConfig": {
+      "columnPartitionMap": {
+        "longDimSV1": {
+          "functionName": "Modulo",
+          "numPartitions": 3
+        }
+      }
+    },
+    "streamConfigs": {
+      "realtime.segment.flush.threshold.size": "63",
+      "realtime.segment.flush.threshold.time": "1h",
+      "streamType": "kafka",
+      "stream.kafka.topic.name": "PinotRealtimeFeatureTest3Event",
+      "stream.kafka.consumer.type": "simple",
+      "stream.kafka.decoder.class.name": "org.apache.pinot.plugin.stream.kafka.KafkaJSONMessageDecoder",
+      "stream.kafka.consumer.factory.class.name": "org.apache.pinot.plugin.stream.kafka20.KafkaConsumerFactory",
+      "stream.kafka.broker.list": "localhost:19092",
+      "stream.kafka.zk.broker.url": "localhost:2181/kafka",
+      "stream.kafka.consumer.prop.auto.offset.reset": "largest"
+    }
+  },
+  "metadata": {
+    "customConfigs": {
+      "d2Name": ""
+    }
+  },
+  "segmentsConfig": {
+    "replicasPerPartition": "1",
+    "replication": "1",
+    "retentionTimeUnit": "",
+    "retentionTimeValue": "",
+    "schemaName": "FeatureTest3",
+    "segmentAssignmentStrategy": "BalanceNumSegmentAssignmentStrategy",
+    "segmentPushFrequency": "daily",
+    "segmentPushType": "APPEND",
+    "timeColumnName": "HoursSinceEpoch",
+    "timeType": "HOURS"
+  },
+  "quota": {
```

Comment:
Do we need QPS quota? We don't want to fail the test because of quota exceed (for whatever reason), so better to remove this.

Reply:
Good catch, removing this.